images/build-e2e/entrypoint.sh

```diff
@@ -6,13 +6,19 @@ if [[ ${PLATFORM} == 'windows' ]]; then
     BINARY=e2e.test.exe
 fi
 BINARY_PATH="/opt/crc/bin/${PLATFORM}-amd64/${BINARY}"
+
+# Running options
+CLEANUP_HOME="${CLEANUP_HOME:-"true"}"
+# Valid options are non-ux or ux
+TESTING_MODE="${TESTING_MODE:-"non-ux"}"
 # Review this when go 1.16 with embed support
 FEATURES_PATH=/opt/crc/features
 TESTDATA_PATH=/opt/crc/testdata
 UX_RESOURCES_PATH=/opt/crc/ux
+
 # Results
 RESULTS_PATH="${RESULTS_PATH:-/output}"
-CLEANUP_HOME="${CLEANUP_HOME:-"true"}"
```

Comment:
This move belongs to the previous commit

Reply:
Changed